CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+# Welcome to Scaffold-ETH 2 Contributing Guide 
+
+Thank you for investing your time in contributing to Scaffold-Eth 2!  
+
+This guide aims to provide overview of the contribution workflow to help us make the contribution process effective for everyone involved. 
+
+## Table of Contents
```

Comment:
I'd love to have a "Vision" or "Philosophy" or something like that to explain about the simple & minimal philosophy, iterations, adding is easier than removing.... and we want to put a lot of intention on it.

Reply:
Added in the about-project section.